jni_gen/cmake/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,30 @@
+# The Flutter tooling requires that developers have CMake 3.10 or later
```

Comment:
Remove this. We're not using this in Flutter apps.

Reply:
`FilesWriter` copies this along with generated C file, because generated library needs to be linked to JVM libraries etc.. depending on the platform. This applies to both flutter and standalone.

Reply:
I'm blind and didn't see the `.tmpl` 🙈